1exercise5.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <title>Task</title>
+    
+    </head>
+<body>
+
+    <script>
+function multiplyNumeric(menu) {
+    for (let key in menu) {
+    
+    if (typeof(menu[key]) === 'number') {
```

Comment:
You don't need brackets when you use `typeof` because `typeof` isn't a function.